fuzzing/common.bzl

```diff
@@ -67,9 +67,7 @@ Rule for creating a script to run the fuzzing test.
 def _fuzzing_corpus_impl(ctx):
     corpus_dir = ctx.actions.declare_directory(ctx.attr.name)
     cp_args = ctx.actions.args()
-
-    for input_file in ctx.files.srcs:
-        cp_args.add(input_file)
+    cp_args.add_all(ctx.files.srcs)
```

Comment:
for my own education, is this supposed to address the comment in the PR description about passing directories in fuzzing_corpus? If so, I don't think there is a current example

Reply:
Added `examples/test_corpus_dir` as a test on directory. 

For now the file structure after copying will be remained since we are using `cp -r`. But after #60 is merged, the file structure will be flattened.